File: cpp programs/prime.cpp
#include <iostream>
using namespace std;
int main()
{
    int n, count = 0;
    cout << "\n enter n";
    cin >> n;
    for (int i = 1; i <= n; i++)
    {
        if (n % i == 0)
        {
            count++;
        }
    }
    if (count == 2)
    {
        cout << "\n prime";
    }
    else
    {
        cout << "\n not prime";
    }
    return 0;
}

File: java programs/DSA/prime.java
public class prime{
    public static void main(String[] args)
    {
        int n=997,count=0;
        for(int i=2;i<=n;i++)
        {
            if(n%i==0)
            {
                count=count+1;
            }
        }
        if(count==1)
        {
            System.out.println("prime");
        }
        else
        {
            System.out.println("no");
        }
    }

}